src/components/ContentUploader/ContentUploader.js

```diff
@@ -68,6 +68,7 @@ type Props = {
     onMinimize?: Function,
     onUpload: Function,
     onUpload: Function,
+    overwrite?: boolean,
```

Comment:
I dont think this needs to be optional, since you are defaulting it to true

Reply:
you're right